(node:1376766) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'banner-nosession-45029',
  containerIp: '10.0.3.118',
  mitmPort: 45029,
  autoAccessFixed: 1
}
2024-11-14 03:21:46.595 - [Info] MITM Version: 2
2024-11-14 03:21:46.598 - [Info] Auto Access Enabled: true
2024-11-14 03:21:46.599 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-14 03:21:46.611 - [Info] SSH man-in-the-middle server for 10.0.3.118 listening on 10.0.3.1:45029
2024-11-14 03:22:26.051 - [Debug] [Connection] Attacker connected: 218.92.0.208 | Client Identification: SSH-2.0-PuTTY
2024-11-14 03:22:27.477 - [Debug] [Auth] Attacker 218.92.0.208 trying to authenticate with 'none'
2024-11-14 03:22:27.478 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2024-11-14 03:22:27.934 - [Debug] [Auth] Attacker 218.92.0.208 trying to authenticate with 'password'
2024-11-14 03:22:27.935 - [Debug] [Auto Access] Attacker: 218.92.0.208, Threshold: 1, Attempts: 1
2024-11-14 03:22:27.936 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Container user is disabled
2024-11-14 03:22:27.938 - [Debug] [Auth] Attacker: 218.92.0.208 has so far made 1 attempts. Remaining: 5 attempts
2024-11-14 03:22:28.392 - [Debug] [Auth] Attacker 218.92.0.208 trying to authenticate with 'password'
2024-11-14 03:22:28.393 - [Debug] [Auto Access] Compromising the honeypot
2024-11-14 03:22:28.393 - [Debug] [Auto Access] Adding the following credentials: 'root:qq147258'
2024-11-14 03:22:28.638 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-14 03:22:28.642 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-14 03:22:28.643 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.118
2024-11-14 03:22:30.056 - [Debug] [LXC] Authentication Failed
2024-11-14 03:22:30.056 - [Debug] [Auth] Attacker authentication error: Error: All configured authentication methods failed
2024-11-14 03:22:30.057 - [Debug] [Auth] Attacker: 218.92.0.208 has so far made 2 attempts. Remaining: 4 attempts
2024-11-14 03:22:30.057 - [Debug] [Auth] Attacker authentication error: Error: All configured authentication methods failed
2024-11-14 03:22:30.057 - [Debug] [Auth] Attacker: 218.92.0.208 has so far made 3 attempts. Remaining: 3 attempts
2024-11-14 03:22:30.064 - [Debug] [LXC] Container's OpenSSH server ended connection
2024-11-14 03:22:30.067 - [Debug] [LXC] Container's OpenSSH server closed connection
2024-11-14 03:22:31.570 - [Debug] [Auth] Attacker 218.92.0.208 trying to authenticate with 'password'
2024-11-14 03:22:31.575 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2024-11-14 03:22:31.577 - [Debug] [Auth] Attacker: 218.92.0.208 has so far made 4 attempts. Remaining: 2 attempts
2024-11-14 03:22:33.516 - [Debug] [Auth] Attacker 218.92.0.208 trying to authenticate with 'password'
2024-11-14 03:22:33.517 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2024-11-14 03:22:33.517 - [Debug] [Auth] Attacker: 218.92.0.208 has so far made 5 attempts. Remaining: 1 attempts
2024-11-14 03:22:34.972 - [Debug] [Auth] Attacker 218.92.0.208 trying to authenticate with 'password'
2024-11-14 03:22:34.972 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2024-11-14 03:22:34.973 - [Debug] [Auth] Attacker: 218.92.0.208 has so far made 6 attempts. Remaining: 0 attempts
2024-11-14 03:22:34.973 - [Debug] [Connection] Max Login Attempts Reached - Closing connection on attacker
2024-11-14 03:22:34.974 - [Debug] [Connection] Attacker closed the connection
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
